lib/buildsystems/python.rb

```diff
@@ -0,0 +1,38 @@
+require 'package'
+
+class Python < Package
+  property :python_build_options
+  property :python_install_options
+  property :no_svem
+
+  depends_on 'python3'
+
+  def self.build
+    @required_pip_modules = %w[build installer setuptools wheel pyproject_hooks]
+    @pip_list = `pip list --exclude pip`
+    @required_pip_modules.each do |pip_pkg|
+      unless @pip_list.include?(pip_pkg)
+        puts "Installing #{pip_pkg} using pip..."
+        system "MAKEFLAGS=-j#{CREW_NPROC} pip install #{pip_pkg}"
+      end
+    end
```

Comment:
Could lines 11-18 be replaced by `depends_on 'pip'`?

Reply:
There is no pip package...  isn't that what `depends_on` looks for?

There is `py3_pip.rb`, but that only installs pip.

Did you mean `buildsystems/pip`? That only checks for those mentioned packages when installing using `pip`.

I think getting those installed with pip is the safest way not to have a broken python package build, so I think it would be better to leave that section as is. Yes it slightly reduplicates code...

Or would it be better to just replace the py3_pip package with something which uses that code, since pip now gets installed with `python3` in any case? Then we could just have `depends_on 'py3_pip'`...

Reply:
Let me see if I can rejigger `py3_pip`...